### ISSUE TITLE
ApiController Error: Invalid child element 'childId'

### ISSUE THREAD

**Issue by ackerdev** [AuthorizeNet/sdk-node#31]
When attempting to make a CreateTransactionRequest, the SDK throws an exception

Example code:
```js
const request = new ApiContracts.CreateTransactionRequest({
  merchantAuthentication: {
    name: constants.apiLoginKey,
    transactionKey: constants.transactionKey
  },
  transactionRequest: {
    transactionType: ApiContracts.TransactionTypeEnum.AUTHCAPTURETRANSACTION,
    amount: 1.00,
    payment: {
      creditCard: {
        cardNumber: '4242424242424242',
        expirationDate: '10/2025'
      }
    }
  }
});

const ctrl = new ApiControllers.CreateTransactionController(request.getJSON());

const ctrl.execute(() => {
  // ...
});
```

Error:
```
Error: The element 'createTransactionRequest' in namespace 'AnetApi/xml/v1/schema/AnetApiSchema.xsd' has invalid child element 'clientId' in namespace 'AnetApi/xml/v1/schema/AnetApiSchema.xsd'.
    // ...
    at Request._callback (/my-project/node_modules/authorizenet/lib/apicontrollersbase.js:115:5)
    at Request.self.callback (/my-project/node_modules/request/request.js:185:22)
    at emitTwo (events.js:126:13)
    at Request.emit (events.js:214:7)
    at Request.<anonymous> (/my-project/node_modules/request/request.js:1157:10)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at IncomingMessage.<anonymous> (/my-project/node_modules/request/request.js:1079:12)
    at Object.onceWrapper (events.js:313:30)
```

**Comment by mkarbowiak** [comment]
Had the same issue. I didn't find anything about it in the docs but forums say:
```
ClientID is for internal use by Authorize.Net and is not required.
```
and
```
The Client ID is an internal value used to identify the SDK client used.
If you set the value to Null, the gateway will not return an error.
```
Setting it to `null` doesn't help (default value set by lib is `null` anyway). It works when I pass request data to the instance using setter methods OR provide some dummy `clientId` value when passing an object to the constructor.

**Comment by ackerdev** [comment]
That's interesting, because my experience was that the `execute` method of the APIController was actually setting clientId to `sdk-node-1.0.4`. Setting clientId to `null` or even to a dummy string only ended with the same result, as the execute method would end up overwriting it.

Instead, I ended up completely forgoing the APIContract classes and passed in an object directly with a no-op setter for clientId, like so:

```js

const request = {
  createTransactionRequest: {
    merchantAuthentication: {
      name: constants.apiLoginKey,
      transactionKey: constants.transactionKey
    },
    transactionRequest: {
      transactionType: ApiContracts.TransactionTypeEnum.AUTHCAPTURETRANSACTION,
      amount: 1.00,
      payment: {
        creditCard: {
          cardNumber: '4242424242424242',
          expirationDate: '10/2025'
        }
      }
    },
    set clientId(clientId) {} // This prevents execute from being able to set clientId at all
  }
};

const ctrl = new ApiControllers.CreateTransactionController(request.getJSON());

const ctrl.execute(() => {
  // ...
});
```

**Comment by gnongsie** [comment]
Hi,

The fix for this has been released as part of the latest version v1.0.7. I will be closing this issue now.

Feel free to reach out to us if any further issues come up.
